apps/drain-safe/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
I know this comes from a template that needs to be updated, but if you have time: can we bump this to es2015? most of the browsers support it nowadays